core/storage/exploration/gae_models.py

```diff
@@ -103,8 +103,6 @@ class ExplorationModel(base_models.VersionedModel):
     # DEPRECATED in v2.0.0.rc.2. Do not use. Retaining it here because deletion
```

Comment:
@michaelw54 This should be deleted too, right?

Reply:
@seanlip done